easybuild/easyblocks/a/abaqus.py

```diff
@@ -126,10 +130,9 @@ def install_step(self):
             # rather than a regular dictionary (where there's no guarantee on key order in general)
             std_qa = OrderedDict()
 
-            # Enable Extended Product Documentation
-            std_qa[selectionstr % (r"\[ \]", "Extended Product Documentation")] = "%(nr)s"
-            # Enable Abaqus CAE (docs)
-            std_qa[selectionstr % (r"\[ \]", "Abaqus CAE")] = "%(nr)s"
+            # Disable Extended Product Documentation (because it has a troublesome Java dependency)
+            std_qa[selectionstr % (r"\[*\]", "Extended Product Documentation")] = "%(nr)s"
+            installed_docs = False
```

Comment:
```suggestion
            # Disable Extended Product Documentation because it has a troublesome Java dependency
            # (and previous support was actually incomplete)
            std_qa[selectionstr % (r"\[*\]", "Extended Product Documentation")] = "%(nr)s"
            installed_docs = False
```